tests/ImageSharp.Tests/TestImages.cs

```diff
@@ -20,6 +20,7 @@ public static class Png
             public const string Blur = "Png/blur.png";
             public const string Indexed = "Png/indexed.png";
             public const string Splash = "Png/splash.png";
+            public const string CorruptedChunk = "Png/big-corrupted-chunk.png";
```

Comment:
We should add a test case as well by extending theory data in `PngDecoderTests`!

Reply:
Will do, good point.